src/mongocxx/collection.hpp

```diff
@@ -15,10 +15,6 @@
 #pragma once
 
 #include <algorithm>
-#include <cstdint>
-#include <memory>
-#include <string>
```

Comment:
Nice. I'm curious, did you use a separate tool to determine these headers were unused? Or did this come from clang-tidy / compiler warnings?

Reply:
We probably _should_ get a tool but CLION just highlights it for me.